From a1c933edfdf3822de6c42b437a1909a62123f1c7 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Wed, 26 Oct 2022 15:40:28 +0200
Subject: [PATCH 118/529] ARM: dts: stm32: change USART1 clock to an SCMI clock
 on stm32mp15 boards

On stm32mp15 boards, change clock of USART1 node to SCMI clock
"CK_SCMI_USART1"

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I9016b0c0d54eadb1e8210ee83e1dac339e0d22c0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277710
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277711
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277712
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277713
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/277714
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157a-ed1-scmi.dtsi | 4 ++++
 arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi | 4 ++++
 arch/arm/boot/dts/stm32mp157c-dk2-scmi.dtsi | 4 ++++
 arch/arm/boot/dts/stm32mp157c-ed1-scmi.dtsi | 4 ++++
 arch/arm/boot/dts/stm32mp157c-ev1-scmi.dtsi | 4 ++++
 5 files changed, 20 insertions(+)

--- a/arch/arm/boot/dts/stm32mp157a-ed1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157a-ed1-scmi.dtsi
@@ -71,3 +71,7 @@
 &rtc {
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
+
+&usart1 {
+	clocks = <&scmi_clk CK_SCMI_USART1>;
+};
--- a/arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi
@@ -74,3 +74,7 @@
 &rtc {
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
+
+&usart1 {
+	clocks = <&scmi_clk CK_SCMI_USART1>;
+};
--- a/arch/arm/boot/dts/stm32mp157c-dk2-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157c-dk2-scmi.dtsi
@@ -77,3 +77,7 @@
 &rtc {
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
+
+&usart1 {
+	clocks = <&scmi_clk CK_SCMI_USART1>;
+};
--- a/arch/arm/boot/dts/stm32mp157c-ed1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157c-ed1-scmi.dtsi
@@ -76,3 +76,7 @@
 &rtc {
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
+
+&usart1 {
+	clocks = <&scmi_clk CK_SCMI_USART1>;
+};
--- a/arch/arm/boot/dts/stm32mp157c-ev1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157c-ev1-scmi.dtsi
@@ -79,3 +79,7 @@
 &rtc {
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
+
+&usart1 {
+	clocks = <&scmi_clk CK_SCMI_USART1>;
+};
